docs(configuration): improve `devServer` documentation (#5490)
webpack/webpack.js.org: src/content/configuration/dev-server.mdx

};
```

> When the server is started, there will be a message before the list of resolved modules:

Reviewer: ```suggestion
When the server is started, there will be a message prior to the list of resolved modules:
```

T> If you're having trouble, navigating to the `/webpack-dev-server` route will show where files are served. For example, `http://localhost:9000/webpack-dev-server`.

> T> If you want to manually recompile the bundle, navigating to the `/invalidate` route will [invalidate](/api/node/#invalidate-watching) the current compilation of the bundle and recompile it for you via [webpack-dev-middleware](https://github.com/webpack/webpack-dev-middleware#invalidate). Depending on your configuration,the URL may look like `http://localhost:9000/invalidate`.

Reviewer: ```suggestion
T> If you want to manually recompile the bundle, navigating to the `/invalidate` route will [invalidate](/api/node/#invalidate-watching) the current compilation of the bundle and recompile it for you via [webpack-dev-middleware](https://github.com/webpack/webpack-dev-middleware#invalidate). Depending on your configuration, the URL may look like `http://localhost:9000/invalidate`.
```

npx webpack serve --no-history-api-fallback
```

> Bypassing an object this behavior can be controlled further using options like `rewrites`:

Reviewer: ```suggestion
By providing an object this behavior can be controlled further using options like `rewrites`:
```

`function (devServer)`

> Provides the ability to execute custom middleware before all other middleware

Reviewer: What's wrong with `prior to`?

Author: Prior to has similar meaning to before ,you can check below.

![prior to - Google Search - Google Chrome 26-09-2021 8 59 08 AM](https://user-images.githubusercontent.com/72331432/134792397-732f2c10-78fe-4752-b5c9-4fef4a8123d9.png)

Reviewer: Then let's stick with `prior to`.

Author: > Then let's stick with `prior to`.

so can you suggest me the changes or i should make a new Pullrequest ?

Reviewer: ```suggestion
Provides the ability to execute custom middleware prior to all other middleware
```

};
```

> T> The browser application name is platform-dependent. Don't hard code it in reusable modules. For example, `'Chrome'` is `'Google Chrome'` on macOS, `'google-chrome'` on Linux , and `'chrome'` on Windows.

Reviewer: ```suggestion
T> The browser application name is platform-dependent. Don't hard code it in reusable modules. For example, `'Chrome'` is `'Google Chrome'` on macOS, `'google-chrome'` on Linux, and `'chrome'` on Windows.
```

`boolean` `string` `[string]` `object` `[object]`

> This option allows configuring options for serving static files from the directory (by default 'public' directory). To disable set it to `false`:

Reviewer: What's wrong with `allows to configure`?

Author: Nothing wrong with allows to configure, this whole sentence is in present continuos tense and allows configuring seems to fit in here that why i did that
